File: app/src/main/java/com/example/rafaelanastacioalves/moby/common/Utility.java
package com.example.rafaelanastacioalves.moby.common;

import android.os.Build;


public class Utility {

    //TODO Implement
    public static String getDeviceBrand(){
        return Build.MANUFACTURER;
    }

    public static String getDeviceModel() {
        return Build.MODEL;
    }

    public static String getAndroidVersion() {
        return Build.VERSION.RELEASE;
    }
}

File: app/src/main/java/com/example/rafaelanastacioalves/moby/common/Utility.kt
package com.example.rafaelanastacioalves.moby.common

import android.os.Build


object Utility {

    //TODO Implement
    val deviceBrand: String
        get() = Build.MANUFACTURER

    val deviceModel: String
        get() = Build.MODEL

    val androidVersion: String
        get() = Build.VERSION.RELEASE
}
